libs/packages/components/src/lib/actions-menu/action-menu.model.ts

```diff
@@ -0,0 +1,21 @@
+export enum ActionMenuMode {
+  SHOWN,
+  DISABLED,
+  HIDDEN,
+}
+
+export interface TriggerTypeModel {
+  type: string;
+  shadow: boolean;
+}
+
+export interface ActionMenuActions {
+  id: string;
+  text: string;
+  mode?: ActionMenuMode;
+}
+
+export interface ActionMenModel {
```

Comment:
ActionMenModel should be ActionMenuModel

Reply:
Update name